linux_docker_resources/rti_web_binaries_install_script.py

```diff
@@ -46,10 +48,34 @@ def install_connext(installer_path, install_directory):
             'Unexpected output while installing Connext. Last 100 chars received from installer: ' +
             '\n' + child.before)
 
+
+def install_plugin(pkg_path, install_directory):
+    child = pexpect.spawn(
+        os.path.join(install_directory, 'bin', 'rtipkginstall') + ' ' + pkg_path, encoding='utf8')
+
+    try:
+        child.expect_exact('Do you want to continue? [Y/n]:')
+        child.sendline('y')
+        child.expect(pexpect.EOF)
+
+    except (pexpect.TIMEOUT, pexpect.EOF):
+        raise RuntimeError(
+            "Unexpected output while installing Connext plugin '%s'." % pkg_path +
+            'Last 100 chars received from installer: ' +
+            '\n' + child.before)
+
+
 if __name__ == '__main__':
     parser = argparse.ArgumentParser()
     parser.add_argument('installer_path')
     parser.add_argument('install_directory')
+    parser.add_argument('--rtipkg_paths', nargs='*', default=[])
 
     args = parser.parse_args()
     install_connext(args.installer_path, args.install_directory)
+    # Install successful, now installing rti plugins if any
+    for plugin_path in args.rtipkg_paths:
+        if os.path.isfile(plugin_path):
+            install_plugin(plugin_path, args.install_directory)
+        else:
+            print("RTI package '%s' not found, Skipping..." % plugin_path, file=sys.stderr)
```

Comment:
Should we enforce that the passed plugins are being installed rather than skip them with just a warning message?

Reply:
Yeah we definitely can do that, I took the conservative approach in case we had some "optional" plugins in the future. I can raise here with the same error message if it's preferred

Reply:
Sounds "safer" to me. If we need optional plugins in the future we can either relax it or pass them using a different option.

Reply:
Sounds good https://github.com/ros2/ci/pull/105/commits/5a6daba2a141d572482e2e534c1d87f21cae54ea